frontend/src/hooks/useRecruitmentItem.js

```diff
@@ -14,7 +14,7 @@ const useRecruitmentItem = (recruitmentId) => {
       setRecruitmentItems(data);
     } catch (error) {
       alert("지원서를 불러오는데 실패했습니다.");
-      history.replace(PATH.HOME);
+      navigate(PATH.HOME);
```

Comment:
```suggestion
      navigate(PATH.HOME, { replace: true });
```

`history.replace`에서 바꾸는 거면 `replace: true`를 같이 줄 것 같은데, 안 준 이유가 있나요~?
여기 말고도 replace 옵션을 줘야 하는 곳들이 있는 것 같아요. 한 번 확인해주세요~

Reply:
수정했습니다~~